eventstore/store_billable_events.go

```diff
@@ -207,7 +204,7 @@ func WithBillableEvents(query string, filter eventio.EventFilter, args ...interf
 				resource_type,
 				org_guid,
 				org_name,
-				quota_definition_guid,
+				null::uuid as quota_definition_guid,
```

Comment:
You may be wondering why `null::uuid`:
* `null` by itself failed some API tests. 
* The empty string failed some consolidating billing tests. 
* This passes both.

Reply:
Okay then